src/components/block-gallery/gallery-image.js

```diff
@@ -175,14 +181,37 @@ class GalleryImage extends Component {
 		return (
 			<figure className={ 'coblocks-gallery--figure ' + className } tabIndex="-1" onKeyDown={ this.onKeyDown } ref={ this.bindContainer }>
 				{ isSelected &&
-					<div className="components-coblocks-remove-gallery-item-button-wrapper">
-						<IconButton
-							icon="no-alt"
-							onClick={ onRemove }
-							className="components-coblocks-remove-gallery-item-button-button"
-							label={ __( 'Remove Image' ) }
-						/>
-					</div>
+					<Fragment>
+						{ supportsMoving &&
+							<div className="components-coblocks-gallery-item__move-menu">
+								<IconButton
+									icon={ verticalMoving ? "arrow-up" : "arrow-left" }
+									onClick={ isFirstItem ? undefined : onMoveBackward }
+									className="coblocks-gallery-item__button"
+									label={ __( 'Move Image Backward' ) }
+									aria-disabled={ isFirstItem }
+									disabled={ ! isSelected }
+								/>
+								<IconButton
+									icon={ verticalMoving ? "arrow-down" : "arrow-right" }
+									onClick={ isLastItem ? undefined : onMoveForward }
```

Comment:
Same here: Instead of using an if-else ternary here we can utilize the `&&` operator and remove the `undefined` return value.

```suggestion
									onClick={ isLastItem && onMoveForward }
```

Reply:
I used `onClick={ ! isLastItem && onMoveForward }`